src/polymer-project.ts

```diff
@@ -112,7 +114,11 @@ export class PolymerProject {
   bundler: Bundler;
 
   constructor(options?: ProjectOptions) {
-    this.root = options.root || process.cwd();
+    this.root = process.cwd();
```

Comment:
btw, this can be just: `this.root = osPath.resolve(options.root)` or perhaps `this.root = osPath.resolve(options.root || '')` to guard against null.